CommunityToolShedMvc/CommunityToolShedMvc/Views/Home/Index.cshtml

```diff
@@ -0,0 +1,32 @@
+﻿@model List<CommunityToolShedMvc.Models.Communites>
+@{
+    ViewBag.Title = "Home";
+}
+
+
+<h2>Home</h2>
+
+<div style="width:35%">
+
+    <table class="table table-sm table-striped table-hover">
+        <thead class="thead-dark">
+            <tr>
+                <th>Name</th>
+                <th>Availability</th>
+                @*<th>Owner</th>*@
+            </tr>
+        </thead>
+        @foreach (var com in Model)
```

Comment:
Naming nitpick... I'd rename `com` to `community` to help improve the readability of the code.